include/fuse_rl/common/sensor_proc.h

```diff
@@ -280,26 +292,10 @@ inline bool processAbsolutePoseWithCovariance(
   // Build the sub-vector and sub-matrices based on the requested indices
   fuse_core::VectorXd pose_mean_partial(position_indices.size() + orientation_indices.size());
   fuse_core::MatrixXd pose_covariance_partial(pose_mean_partial.rows(), pose_mean_partial.rows());
-  const size_t base_ind_pos = static_cast<size_t>(fuse_variables::Position2DStamped::X);
-  const size_t base_ind_or = static_cast<size_t>(fuse_variables::Orientation2DStamped::YAW);
 
-  appendPartialMeasurement(
-    pose_mean,
-    pose_covariance,
-    position_indices,
-    base_ind_pos,
-    0u,
-    pose_mean_partial,
-    pose_covariance_partial);
-
-  appendPartialMeasurement(
-    pose_mean,
-    pose_covariance,
-    orientation_indices,
-    base_ind_or,
-    position->size(),
-    pose_mean_partial,
-    pose_covariance_partial);
+  const auto indices = mergeIndices(position_indices, orientation_indices, position->size());
+
+  appendPartialMeasurement(pose_mean, pose_covariance, indices, pose_mean_partial, pose_covariance_partial);
```

Comment:
I'm going to claim this is really a `populate` operation, rather than an `append` operation. Since the `mean_partial` index always starts from zero. I do note that this PR doesn't actually change that aspect of the function, but as long as we are modifying it, might as well make the name describe the operation.

Reply:
That's a very good point and I agree it makes sense to rename it to `populatePartialMeasurement`.

If @ayrton04 also agrees on that, I can do that change.

Reply:
Sure